target/linux/ath79/base-files/etc/board.d/02_network

```diff
@@ -94,31 +88,23 @@ ath79_setup_interfaces()
 		;;
 	"tplink,tl-wr1043nd-v2"|\
 	"tplink,tl-wr1043nd-v3")
-		ucidef_set_interfaces_lan_wan "eth1.1" "eth0.2"
 		ucidef_add_switch "switch0" \
 			"0@eth1" "1:lan:4" "2:lan:3" "3:lan:2" "4:lan:1" "5:wan" "6@eth0"
 		;;
 	"tplink,tl-wr2543-v1")
 		ucidef_add_switch "switch0" \
 			"1:lan" "2:lan" "3:lan" "4:lan" "0:wan" "9@eth0"
 		;;
-	"netgear,wndr3800")
-		ucidef_set_interfaces_lan_wan "eth0" "eth1"
-		ucidef_add_switch "switch0" \
-			"0:lan:4" "1:lan:3" "2:lan:2" "3:lan:1" "5u@eth0"
-
-		ucidef_add_switch_attr "switch0" "blinkrate" 2
-		ucidef_add_switch_port_attr "switch0" 1 led 6
-		ucidef_add_switch_port_attr "switch0" 2 led 9
-		ucidef_add_switch_port_attr "switch0" 5 led 2
-		;;
-	"buffalo,wzr-hp-g450h")
+	"tplink,tl-wr740n-v2")
+		ucidef_set_interfaces_wan "eth0"
```

Comment:
(Also here: changing from ucidef_set_interfaces_lan_wan "eth1.1" "eth0" to ucidef_set_interfaces_wan "eth0" by intention?)

Reply:
Yes. LAN should be handled by  ucidef_add_switch. (And I forgot this also...)